utils/nanoaod/triggers.py

```diff
@@ -0,0 +1,215 @@
+trigger_table = {
```

Comment:
The triggers in NanoAOD are not stored as an array, I don't see why this is needed.

Reply:
removed